src/platform/packages/shared/react/kibana_context/root/eui_provider.tsx

```diff
@@ -36,28 +36,34 @@ export interface KibanaEuiProviderProps extends Pick<EuiProviderProps<{}>, 'modi
   globalStyles?: boolean;
 }
 
-// Set up the caches.
-// https://eui.elastic.co/#/utilities/provider#cache-location
-const stylisPlugins = [euiStylisPrefixer]; // https://emotion.sh/docs/@emotion/cache#stylisplugins
+
+const sharedCacheOptions = {
+  // Set up the caches.
+  // https://eui.elastic.co/#/utilities/provider#cache-location
```

Comment:
While you're at it, could you please update the link to point to the new EUI docs site?

```suggestion
  // https://eui.elastic.co/docs/utilities/provider/#emotioncache-customization
```